src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticAnalysisContextHelpers.cs

```diff
@@ -70,6 +70,14 @@ internal static void VerifyDiagnosticLocationsInCompilation(Diagnostic diagnosti
                 }
             }
         }
+        
+        internal static void VerifyIOperationFeatureFlag(bool isIOperationFeatureEnabled)
```

Comment:
I presume this is mostly the functionality we had for Update3, remove it in master and are now adding it back?